tb_plugin/ci_scripts/install_env.sh

```diff
@@ -24,9 +24,10 @@ pip install numpy tensorboard typing-extensions pillow pytest
 if [ "$PYTORCH_VERSION" = "nightly" ]; then
     pip install --pre torch -f "https://download.pytorch.org/whl/nightly/$CUDA_VERSION/torch_nightly.html"
     pip install --pre torchvision --no-deps -f "https://download.pytorch.org/whl/nightly/$CUDA_VERSION/torch_nightly.html"
-elif [ "$PYTORCH_VERSION" = "1.8rc" ]; then
-    pip install --pre torch -f "https://download.pytorch.org/whl/test/$CUDA_VERSION/torch_test.html"
-    pip install --pre torchvision --no-deps -f "https://download.pytorch.org/whl/test/$CUDA_VERSION/torch_nightly.html"
```

Comment:
we may still need `1.8rc` for a few more days...

Reply:
Thx, done.